### PR TITLE
Update webpack configuration

### PR DESCRIPTION
**Pull request by antipin**
* Bring autoprefixer back
* Remove hash appending to classnames for dev env

### CODE REVIEW

File: etc/webpack/utils/utils.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-function getRuleJS(useLinting = true, ...includePaths) {
+const postcssAutoprefixer = require('autoprefixer'); // eslint-disable-line import/no-extraneous-dependencies
+
+function getRuleJS({ useLinting = true, includePaths }) {
     const loaders = useLinting ? ['babel-loader', 'eslint-loader'] : ['babel-loader'];
```
Comment (eszthoff) [inline etc/webpack/utils/utils.js]: this is not needed

File: etc/webpack/webpack.config.js [hunk 2]
```diff
@@ -27,34 +26,16 @@ const plugins = {
...
-            {
-                loader: 'sass-loader',
-                options: {
-                    includePaths: [SOURCE_PATH]
-                }
-            }
-        ]
-    }
+    js: getRuleJS({
+        useLinting: env === 'prod',
```
Comment (eszthoff) [inline etc/webpack/webpack.config.js]: no need to set this

File: etc/webpack/utils/utils.js (hunk 1 above)
Comment (antipin) [reply etc/webpack/utils/utils.js]: done

File: etc/webpack/webpack.config.js (hunk 2 above)
Comment (antipin) [reply etc/webpack/webpack.config.js]: done

**Pull request merged**